python/pyspark/pandas/frame.py

```diff
@@ -5317,6 +5317,12 @@ def to_orc(
         ...     '%s/to_orc/foo.orc' % path,
         ...     mode = 'overwrite',
         ...     partition_cols=['date', 'country'])
+
+        Notes
+        -----
+        pandas-on-Spark writes ORC files into the directory, `path`, and writes
+        multiple `part-...` files in the directory unlike pandas.
```

Comment:
```suggestion
        multiple part files in the directory unlike pandas.
```

Reply:
Out of curiosity, is "part files" a common term?